avalon_client/setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Intel Corporation
```

Comment:
2019 -> 2020

Reply:
Can we rename avalon_client directory to avalon_client_sdk. Do we need avalon_client directory ?

Reply:
Yes, to package all of the python modules into avalon_client_sdk package.

Reply:
avalon_client can have a different interpretation like clients for avalon like echo_clinet, generic_client, heart_disease_eval etc. May be we can rename top level directory as avalon_client_sdk and inside that we can create an sdk directory for packaging python modules. or top level directory as sdk and inside that avalon_client_sdk and avalon_connector_sdk.